src/extract.js

```diff
@@ -50,11 +50,23 @@ exports.TranslationReference = class TranslationReference {
 };
 
 function preprocessScriptTags(data, type) {
+  if (type !== 'vue') {
+    return {};
+  }
```

Comment:
this should be checked before, not after entering the function. Maybe call the function preprocessVueFile and only go there if you actually have a Vue file.

Reply:
I moved out the check for `type` from `preprocessScriptTags` at 9895e63.

Then the function name was changed to `preprocessVueFile` at 5884897.